frontend/packages/schema-editor/src/types/index.ts

```diff
@@ -28,6 +28,9 @@ export enum NameError {
 
 export enum NumberRestrictionsError {
   NoError = 'NoError',
-  InvalidValue = 'InvalidValue',
-  InvalidMaxMinValue = 'InvalidMaxMinValue'
+  Both_max_and_min_inclusive = "Both_max_and_min_inclusive",
+  Integer_one_inclusive_other_exclusive = "Integer_one_inclusive_other_exclusive",
+  Integer_both_are_exclusive = "Integer_both_are_exclusive",
+  Decimal_One_value_should_be__exclusive = "Decimal_One_value_should_be__exclusive",
+
```

Comment:
Har noen forslag til navn som passer bedre med feilmeldingen som skal vises:
```suggestion
  MinMustBeLessThanOrEqualToMax = "MinMustBeLessThanOrEqualToMax",
  MinMustBeLessThanMax = "MinMustBeLessThanMax",
  IntervalMustBeLargeEnough = "IntervalMustBeLargeEnough",
```

Reply:
Fikset